packages/core/graph/package.json

```diff
@@ -20,6 +20,7 @@
     "node": ">= 12.0.0"
   },
   "dependencies": {
+    "@parcel/utils": "^2.9.3",
```

Comment:
We once had a PR that removed this dependency to get the graph package working in more environments: https://github.com/parcel-bundler/parcel/pull/8630

Reply:
Maybe make the BitSet utils into a separate package?

Reply:
Hmm I had not considered that others would be using our graph package. I guess we would have to release this as a major then (changing the type of nodes)... 😕

I'd really like to avoid yet another package. We were just talking about consolidating them haha. Any other ideas?

Reply:
Ok, I moved BitSet into the graph package, and made the default bundler use it from there instead of utils.